docker/requirements.txt

```diff
@@ -0,0 +1,6 @@
+matplotlib
```

Comment:
When installing d3rlpy, the dependencies are also automatically installed. So you don't need this file.